docs/features.md

```diff
@@ -0,0 +1,48 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.MsBuild addin.
+---
+The [Cake.Issues.MsBuild addin] provides the following features.
+
+# Basic features
+
+* Reads warnings from MSBuild log files.
+* Provides URLs for all code analysis (`CA*`) and StyleCop (`SA*`) warnings.
+* Support for custom URL resolving using the [MsBuildAddRuleUrlResolver] alias.
+
+# Supported log file formats
+
+* [MsBuildBinaryLogFileFormat] alias for reading issues from binary log files.
+* [MsBuildXmlFileLoggerFormat] alias for reading issues from log files created by [MSBuild Extension Pack XmlFileLogger].
+
+# Supported comment formats
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                |
+
+# Supported IIssue properties
```

Comment:
[MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.